docs/blog/100days/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 100 Days Of Gatsby - Our Guided Coding Challenge for 2020
+date: 2019-12-20
+author: "Hashim Warren"
+tags: ["learning-to-code", "contest"]
+---
+
+import EmailCaptureForm100Days from "../../../www/src/components/email-capture-form-100days"
+
+### If you're looking for a fun way to explore how to build blazing fast websites with Gatsby, our #100DaysOfGatsby challenge is for you!
+
+![alt text](/100daysofgatsby.png "100 Days of Gatsby")
+
+We’re kicking off this “100 Days” challenge on January 1st, so why not start the year with a Gatsby project?
+
+Udemy named Gatsby [the #1 emerging tech skill](https://www.cnbc.com/2019/12/02/10-hottest-tech-skills-that-could-pay-off-most-in-2020-says-new-report.html) for 2020. We think that's because using Gatsby doesn't just give you a highly performant website, but [learning the framework](/blog/2019-05-03-how-gatsby-helped-jump-start-my-engineering-career/) can help you master [React](/docs/glossary/react/), [NodeJS](/docs/glossary/node/), [GraphQL](/docs/glossary/graphql/), and other modern web technologies.
+
+Register below to join the #100daysofGatsby coding challenge!
+
+<EmailCaptureForm100Days signupMessage="Get weekly updates on new challenges, ideas for where to start, and step-by-step documentation for completing the challenge." />
```

Comment:
should this be renamed more towards the 100days challenge and for daily updates? i think with the 100Days i get 1 email per day?